shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -627,4 +672,32 @@ - (void)scrollWheel:(NSEvent*)event {
   [self dispatchMouseEvent:event phase:kHover];
 }
 
+int MapFile(const char* inPathName, void** outDataPtr, size_t* outDataLength, bool executable) {
```

Comment:
This should have internal linkage (so, `static`). It also needs a declaration comment, and I'd prefer it be at the start of the file (just below kDefaultWindowFramebuffer, outside the class) since that's a common convention for static functions (to avoid declaration order issues).